api/proto/metrics/metrics.proto

```diff
@@ -144,6 +145,48 @@ message appMetric {
   repeated appDiskMetric disk = 6;
 }
 
+message logMetric {
+  uint64 numDeviceEventsSent = 1;
+  uint64 numDeviceBundlesSent = 2;
+  uint64 numAppEventsSent = 3;
+  uint64 numAppBundlesSent = 4;
+  // num4xxResponses: This counter is akin to event errors/drops.
+  //
+  // 4xx response for a device event bundles does not make sense, unless
+  // there is a programming error in EVE code that send device event bundles to cloud.
+  // But, app instances can disappear from cloud and any futher logs sent by EVE to cloud
+  // will result in a 4xx response back.
+  // Any time an app log bundles get rejected with 4xx response from cloud, num4xxResponses
+  // should be incremented by the number of events present in that bundle.
+  //
+  // totalAppLogInput = numAppEventsSent + numAppEventErrors + num4xxResponses
+  uint64 num4xxResponses = 5;
+  google.protobuf.Timestamp lastDeviceBundleSendTime = 6;
+  google.protobuf.Timestamp lastAppBundleSendTime = 7;
+
+  bool isLogProcessingDeferred = 8;
+  uint64 numTimesDeferred = 9;
+  google.protobuf.Timestamp lastLogDeferTime = 10;
+
+  // Quick increments in these counters may mean that
+  // logmanager is experiencing a lot of pressure from rsyslog.
+  // Log bundles are getting filled rapidly.
+  uint64 numDeviceBundleOverFlows = 11;
+  uint64 numAppBundleOverFlows = 12;
+
+  // totalDeviceLogInput = numDeviceEventsSent + numDeviceEventErrors
```

Comment:
No need to repeat this if you put it up top.

And it might benefit from some english text such as
We track device and app logs separately in most of these counters (XXX or all?)
An event is counter as LogInput. That can be sent onwards or dropped for a few reasons. Thus:
totalDeviceLogInput = numDeviceEventsSent + numDeviceEventErrors
totalAppLogInput = numAppEventsSent + numAppEventErrors + num4xxResponses

Reply:
Done. Move as suggested and added comments above.